packages/main/src/ListItem.js

```diff
@@ -123,6 +123,23 @@ const metadata = {
 		_focused: {},
 		"_selection-requested": {},
 	},
+	managedSlots: true,
+	slots: /** @lends sap.ui.webcomponents.main.ListItem.prototype */ {
+
+		/**
+		 * Defines a <code>ui5-button</code> which serves as a replacement of
```

Comment:
[minor remark] in general, button HTML element could be also used in the slot, although would not look nice. So, maybe a bit different wording will be better, as an example:
```js
/**
* Defines the delete button, displayed in "Delete" mode.
* <b>Note:</b> While the slot allows custom buttons, to match
design guidelines, please use the <code>ui5-button</code> component.
* <b>Note:</b> When the slot is not present, a built-in delete button will be displayed.
*/
```

Reply:
Thanks for the improvement idea.